### ISSUE TITLE
Implement support for OL8 / podman

### ISSUE THREAD

**Issue by jmichler** [oracle/docker-images#1376]
With Oracle Linux 8 / RHEL8 the "default" container engine is not docker but podman / buildah. While this seems to be compatible with the scripts here some guidance from oracle on
a) is this a certified/supported combination
b) minor modifications on the buildImage.sh scripts to avoid "wrong" warnings on version checks
would be appreciated.

**Comment by Djelibeybi** [comment]
a) Yes, it is. 
b) Yes, updating the `buildImage.sh` scripts should be done. However, as no actual Oracle products are certified to _run_ on OL8 yet, it doesn't have to happen today.

**Comment by mark-au** [comment]
For reference: using an alias of docker to podman will cause ignorable errors in some `buildDockerImage.sh` scripts as podman currently doesn't report the _Server_ field.

```
$ podman version --format '{{.Server.Version | printf "%.5s" }}'
Error: template: image:1:9: executing "image" at <.Server.Version>: can't evaluate field Server in type libpod.Version
```

This results in
```
$ ./buildDockerImage.sh -v 12.2.0.1 -s
Error: template: image:1:9: executing "image" at <.Server.Version>: can't evaluate field Server in type libpod.Version
./buildDockerImage.sh: line 58: [: : integer expression expected
Checking if required packages are present and valid...
md5sum: linuxx64_12201_database.zip: No such file or directory
```

**Comment by mark-au** [comment]
@Djelibeybi, How did we want to address this? 
We could create an example function that resolves the podman issue and pass it to all the maintainers to introduce or we could update all `buildImage.sh` ourselves with many PRs and have them each review the changes?

**Comment by Djelibeybi** [comment]
How many of the `buildImage.sh` files are affected? Can you point me towards the offending bit in the script that Podman doesn't support?

**Comment by mark-au** [comment]
There are 26 `buildDockerImage.sh` files and I incorrectly assumed it would be many. It's only one: https://github.com/oracle/docker-images/blob/master/OracleDatabase/SingleInstance/dockerfiles/buildDockerImage.sh#L54
The error exists because podman doesn't report separate Server and Client objects

```
$ docker version -f '{{.Server.Version}}'
19.03.1-ol
$ podman version -f '{{.Server.Version}}'
Error: template: image:1:9: executing "image" at <.Server.Version>: can't evaluate field Server in type libpod.Version
```

It looks like this has been resolved in upstream libpod in December and slated to be in a future v1.7 release (https://github.com/containers/libpod/pull/4608)

We could introduce a temporary fix that catches the failure and tries podman syntax but right now ./OracleDatabase/SingleInstance/dockerfiles/buildDockerImage.sh continues after the error so there is no impact to builders. That said, we should not continue after comparison errors and that should be fixed.

**Comment by Djelibeybi** [comment]
Even if it gets fixed in a newer version, that won't help when you run the script on the system with an older version. I'd probably just test to see if `podman` was installed at all and then bypass the version check (assuming the build works ok with Podman as released on OL8).

**Comment by Djelibeybi** [comment]
Fixed in #1494
